### PR TITLE
Offer completion for protected after private

### PR DESCRIPTION
**Pull request by jcouv**
I did a pass on IDE scenarios for the "private protected" feature. So far, this is the only issue I found.

Fixes https://github.com/dotnet/roslyn/issues/20810

@CyrusNajmabadi @dotnet/roslyn-ide for review.
FYI @gafter

### CODE REVIEW

File: src/Features/CSharp/Portable/Completion/KeywordRecommenders/ProtectedKeywordRecommender.cs [hunk 1]
```diff
@@ -62,12 +62,11 @@ private static bool IsValidContextForType(CSharpSyntaxContext context, Cancellat
 
         private static bool CheckPreviousAccessibilityModifiers(CSharpSyntaxContext context)
         {
-            // We can show up after 'internal'.
+            // We can show up after 'internal' and 'private'.
```
Comment (jasonmalinowski) [inline src/Features/CSharp/Portable/Completion/KeywordRecommenders/ProtectedKeywordRecommender.cs]: What about the other direction (i.e. protected private)?

Comment (jcouv) [inline src/Features/CSharp/Portable/Completion/KeywordRecommenders/ProtectedKeywordRecommender.cs]: Good catch, thanks.
Pushed a commit to address that. Per discussion with Neal, we should do the same as with "internal protected" (ie. allow it in either order).

Comment (jcouv) [inline src/Features/CSharp/Portable/Completion/KeywordRecommenders/ProtectedKeywordRecommender.cs]: @jasonmalinowski Could you take another look?

**Comment by jcouv** [comment]
test windows_release_vs-integration_prtest please

**Pull request merged**